development.md

```diff
@@ -0,0 +1,95 @@
+# Development Policies
+
+In this document we introduce some development policies, such as:
+
+- Do not commit directly to master branch. 
+- Create a new branch.
+- Make a pull request.
+- Add two colleagues as reviewers.
+- Run all tests before creating a pull request.
+
+## Coding
+
+### Code style
+
+You must follow good code styles. 
```

Comment:
We must be more specific in this part. In a future improvement of this document, some additional tips and recommendations could be added.